app/views/layouts/_navbar.html.erb

```diff
@@ -123,7 +123,7 @@
   <% if @next_event %>
     <div class="hidden-sm hidden-xs hidden-md" id="next-event" data-event-time="<%= @next_event.next_occurrence_time_attr.to_datetime %>"
          data-event-url="<%= event_path(@next_event) %>"
-         data-event-name="<%= @next_event.name %>"></div>
+         data-event-name="<%= current_user ? @next_event.name : 'Next projects review meeting - listen in to learn more about AgileVentures' %>"></div>
```

Comment:
this could also be replaced by a helper, but text needs to be slightly different for this countdown ...